cmds/boot/systemboot/main.go

```diff
@@ -18,6 +19,124 @@ import (
 	"github.com/u-root/u-root/pkg/smbios"
 )
 
+const (
+	_FB_OEM_IPMI_NET_FUNCTION = 0x36
+
+	_FB_OEM_SET_PROC_INFO = 0x10
+	_FB_OEM_SET_DIMM_INFO = 0x12
+
+	_WIWYNN_ID0     = 0x0
+	_WIWYNN_ID1     = 0x9c
+	_WIWYNN_ID2     = 0x9c
+	_MFGID_AENEON   = 0x5705
```

Comment:
I don't think this should be here. How about having a separate package with all the vendor identifiers? Or maybe inside the `smbios` package.

Reply:
How about adding a file like "Oem.go" into systemboot folder? We can implement all OEM features in the file.

Reply:
The map of manufacturer id -> manufacturer name may be useful for other stuff as well such as dmidecode. I think it would e useful to have it in a package and not inside cmds/boot directory.

Reply:
I moved the vendor identifiers to pkg/smbios/type17_memory_device.go.